packages/synthetic-chain/cli.ts

```diff
@@ -4,10 +4,13 @@ import { parseArgs } from 'node:util';
 import path from 'node:path';
 import { execSync } from 'node:child_process';
 import { buildProposalSubmissions, buildTestImages } from './src/cli/build.js';
-import { refreshDockerfile } from './src/cli/dockerfileGen.js';
+import { writeDockerfile } from './src/cli/dockerfileGen.js';
 import { matchOneProposal, readProposals } from './src/cli/proposals.js';
 import { debugTestImage, runTestImages } from './src/cli/run.js';
 
+// TODO change the tag to 'main' after multi-platform support https://github.com/Agoric/agoric-3-proposals/pull/32
+const baseImage = 'pr-32-linux_arm64_v8';
```

Comment:
This looks like a tag, not an image

Reply:
Good point. It had included the full image path but I simplified it to a tagname. I'll update.